cactus/plugin/builtin/pagecontext.py

```diff
@@ -0,0 +1,120 @@
+#coding:utf-8
+import logging
+import re
+
+
+logger = logging.getLogger(__name__)
+
+
+class PageContextPlugin(object):
```

Comment:
But this "contract" would mean that context handling would not be pluggable.

Certainly you are right, this plugin has to have a low `ORDER`. Is there any documentation or agreement on the expected / desirable values for `ORDER` for different cases?